main.js

```diff
@@ -168,7 +168,7 @@ bot.addListener("message", function(from, to, text, message) {
 		return;
 	}
 	if(splitup[0].toLowerCase() == "..help" || splitup[0].toLowerCase() == "!help" || splitup[0].toLowerCase() == ".help"){
-		bot.say(config.channels[0], "..ping, ..sing, ..insult [name], ..dab [name], ..tell nick message, ..sotd [link], ..lifts [who] [lift] [[weight units reps]], ..$ [currency]");
+		bot.say(config.channels[0], "..ping, ..sing, ..insult [name], ..dab [name], ..tell nick message, ..sotd [link], ..lifts [who] [lift] [[weight units reps]], ..weather [place], [country abbreviation], ..$ [value] [currency code]");
```

Comment:
weather requires both a country and a place, and ..$ requires both the amount and the currency code

Reply:
true

Reply:
i have no idea how to do that

Reply:
also since i put the commas outside the brackets, will it break everything?

Reply:
brackets indicate optional parameters

Reply:
okay... so no brackets indicate mandatory parameters?

Reply:
that's right